src/app/ApplicationTemplate.Business/ForcedUpdates/UpdateRequiredServiceMock.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Reactive.Linq;
+using ApplicationTemplate.DataAccess;
+
+namespace ApplicationTemplate.Business;
+
+/// <summary>
+/// Mock implementation of the <see cref="IUpdateRequiredService"/>.
+/// </summary>
+public sealed class UpdateRequiredServiceMock : IUpdateRequiredService
```

Comment:
This is named like a mock implementation, but I'm not sure why. It looks at the data from the repository and reports on it. I think the only fake implementation is the repository, but this is transparent for this service.